client/app/components/form/UserOperatorReviewForm.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import { useParams } from "next/navigation";
+
+import { userOperatorInternalUserUiSchema } from "@/app/utils/jsonSchema/userOperator";
+import MultiStepAccordion from "@/app/components/form/MultiStepAccordion";
+import { UserOperatorFormData } from "@/app/components/form/formDataTypes";
+import { RJSFSchema } from "@rjsf/utils";
+import UserOperatorReview from "@/app/components/routes/access-requests/form/UserOperatorReview";
+
+interface Props {
+  formData: UserOperatorFormData;
+  schema: RJSFSchema;
+}
+
+const UserOperatorReviewForm = ({ formData, schema }: Props) => {
+  const params = useParams();
+  const userOperatorId = params.id;
+  const isNewOperator = formData.is_new;
+  const userOperatorStatus = formData.status;
+
+  return (
+    <MultiStepAccordion
+      schema={schema}
+      uiSchema={userOperatorInternalUserUiSchema}
+      formData={formData}
+      // Add Review components to the beforeForm prop
+      beforeForm={{
+        "Operator Information": isNewOperator && (
+          <UserOperatorReview
+            userOperator={formData as UserOperatorFormData}
+            userOperatorId={Number(userOperatorId)}
+            note="This is a new operator. You must approve this operator before approving its admin."
+            isOperatorNew={formData?.is_new}
+            operatorId={formData?.operator_id}
+            showRequestChanges={false}
+          />
+        ),
+        "User Information": (
+          <UserOperatorReview
+            userOperator={formData as UserOperatorFormData}
+            userOperatorId={Number(userOperatorId)}
+            operatorId={formData?.operator_id}
+            showRequestChanges={false}
+          />
+        ),
+      }}
+      // If the operator is new, the first section should be expanded
+      // If the user is pending, the second section should be expanded
+      expandedSteps={{
+        "Operator Information": isNewOperator,
+        "User Information": userOperatorStatus === "Pending",
+      }}
+    />
```

Comment:
Note: this was not in the acceptance criteria so happy to remove this if you think it's a bad idea. This expands each section by default if we are displaying the review component.

Reply:
IMO, it's a great idea!